### PR TITLE
Added ':' to clock 7

### PR DESCRIPTION
**Pull request by waielal**

### CODE REVIEW

File: sketch.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 window.addEventListener('load', function() {
   var clocks = new Array(12).fill(0).map(function(_, i) {
     i++;
-    var rendersEveryFrame = [1, 3, 6, 8, 9, 11].indexOf(i) >= 0;
```
Comment (meiamsome) [inline sketch.js:4]: Why did you remove clock 9 here?

Comment (waielal) [reply sketch.js:4]: Because I thought that it didn't need to be updated every frame.

Comment (meiamsome) [reply sketch.js:4]: I believe that there are mini clocks on it that have a second... oh, right yep I got that wrong! Thought I was working on updating every minute here. *whistles*

Comment (waielal) [reply sketch.js]: It's okay :sweat_smile: 

But I'm not quite sure if it was a great idea that I added clock 7 to the list.
In the original implementation the clock updates twice a second and updating it every frame is a little bit over kill.
We could change the implemetation to update only once a second to increase perfomance.
What is your opinion on that?
